core/src/main/scala/sttp/tapir/server/interpreter/BodyListener.scala

```diff
@@ -0,0 +1,13 @@
+package sttp.tapir.server.interpreter
+
+import scala.util.Try
+
+trait BodyListener[F[_], B] {
+  def onComplete(body: B)(cb: Try[Unit] => F[Unit]): F[B]
+}
+
+object BodyListenerSyntax {
```

Comment:
I think if you add the implicit in the companion object (`BodyListener`), then you won't have to import the syntax, it will be automatically searched

Reply:
I did, and it still requires both
```
import sttp.tapir.server.interpreter.BodyListener
import sttp.tapir.server.interpreter.BodyListener._
```

Reply:
ah, I thought that this is in the implicit scope. But apparently not :)